.github/workflows/playtest.yml

```diff
@@ -0,0 +1,65 @@
+name: battlecode-playtest
```

Comment:
This is the github action file. Which should be pretty self documenting because of all the names. I've only scraped the surface of what github actions can do. The editor on the web guides you as you make one, and they have a ton of pre-made examples for anyone to use.